README.md

```diff
@@ -52,6 +52,22 @@ application should be configured to be a **Regular Web Application**.
 * `AUTH0_DOMAIN`: The Auth0 application's domain
 * `AUTH0_KEY`: The Auth0 application's client id
 * `AUTH0_SECRET`: The Auth0 application's client secret.
+* `TWILIO_ACCOUNT_SID`: Twilio Account SID for client
+* `TWILIO_AUTH_TOKEN`: Twilio Auth Token for client
+* `TWILIO_SMS_NUMBER`: Phone number received from Twilio for client
+
+## Running NGROK
+NGROK is needed to manually test sending text messages
```

Comment:
You should expand on this since `ngrok` by itself doesn't have much to do with sending text messages. `ngrok` is used to create IP tunnels from a host behind a firewall to the public internet, not really much to do with sending and receiving text messages.

I wrote this for my own documentation when I was updating your code. Perhaps you can get some inspiration from it?

* https://github.com/ErrorsAndGlitches/waisn-tech-tools/tree/2-sns-integration#setting-up-twilio